arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/Ior.kt

```diff
@@ -263,10 +263,7 @@ public sealed class Ior<out A, out B> {
     return fold({ f(c, it) }, { g(c, it) }, { a, b -> g(f(c, a), b) })
   }
 
-  @Deprecated(
-    NicheAPI + "Prefer when or fold instead",
-    ReplaceWith("  MN.run { fold({ f(it) },{ g(it) },{ a,b -> f(a).combine(g(b)) })}")
-  )
+  @Deprecated(NicheAPI + "Prefer when or fold instead. See the Arrow web migration guide for more info.")
```

Comment:
Have we created a task for adding these changes to the migration guide?

Reply:
I've just created it 😄 . Thanks for the suggestion, @franciscodr!